url7606378bcc_ljy2010a_protobuf2_tgz-pJ8-com_game_gameObj_bufBuilder_ReflectBuilderJ8/src/com/game/protobuf/GameProto.java

```diff
@@ -0,0 +1,7112 @@
+    public static final int SRVTIMER_FIELD_NUMBER = 4;
+    private long srvTimer_;
+    public boolean hasSrvTimer() {
+      return ((bitField0_ & 0x00000008) == 0x00000008);
+    }
+    public long getSrvTimer() {
+      return srvTimer_;
+    }
+    
+    // optional uint32 msgSN = 5;
+    public static final int MSGSN_FIELD_NUMBER = 5;
+    private int msgSN_;
+    public boolean hasMsgSN() {
+      return ((bitField0_ & 0x00000010) == 0x00000010);
+    }
+    public int getMsgSN() {
+      return msgSN_;
+    }
+    
+    // optional .protobuf.UserInfo sender = 6;
+    public static final int SENDER_FIELD_NUMBER = 6;
+    private com.game.protobuf.GameProto.UserInfo sender_;
+    public boolean hasSender() {
+      return ((bitField0_ & 0x00000020) == 0x00000020);
+    }
+    public com.game.protobuf.GameProto.UserInfo getSender() {
+      return sender_;
+    }
+    public com.game.protobuf.GameProto.UserInfoOrBuilder getSenderOrBuilder() {
+      return sender_;
+    }
+    
+    // optional .protobuf.Request request = 7;
+    public static final int REQUEST_FIELD_NUMBER = 7;
+    private com.game.protobuf.GameProto.Request request_;
+    public boolean hasRequest() {
+      return ((bitField0_ & 0x00000040) == 0x00000040);
+    }
+    public com.game.protobuf.GameProto.Request getRequest() {
+      return request_;
+    }
+    public com.game.protobuf.GameProto.RequestOrBuilder getRequestOrBuilder() {
+      return request_;
+    }
+    
+    // optional .protobuf.Response response = 8;
+    public static final int RESPONSE_FIELD_NUMBER = 8;
+    private com.game.protobuf.GameProto.Response response_;
+    public boolean hasResponse() {
+      return ((bitField0_ & 0x00000080) == 0x00000080);
+    }
+    public com.game.protobuf.GameProto.Response getResponse() {
+      return response_;
+    }
+    public com.game.protobuf.GameProto.ResponseOrBuilder getResponseOrBuilder() {
+      return response_;
+    }
+    
+    // optional .protobuf.Notice Notice = 9;
+    public static final int NOTICE_FIELD_NUMBER = 9;
+    private com.game.protobuf.GameProto.Notice notice_;
+    public boolean hasNotice() {
+      return ((bitField0_ & 0x00000100) == 0x00000100);
+    }
+    public com.game.protobuf.GameProto.Notice getNotice() {
+      return notice_;
+    }
+    public com.game.protobuf.GameProto.NoticeOrBuilder getNoticeOrBuilder() {
+      return notice_;
+    }
+    
+    private void initFields() {
+      msgLength_ = 0;
+      msgType_ = "";
+      senderTimer_ = 0L;
+      srvTimer_ = 0L;
+      msgSN_ = 0;
+      sender_ = com.game.protobuf.GameProto.UserInfo.getDefaultInstance();
+      request_ = com.game.protobuf.GameProto.Request.getDefaultInstance();
+      response_ = com.game.protobuf.GameProto.Response.getDefaultInstance();
+      notice_ = com.game.protobuf.GameProto.Notice.getDefaultInstance();
+    }
+    private byte memoizedIsInitialized = -1;
+    public final boolean isInitialized() {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The cyclomatic complexity of this method is 12. By comparison, 98% of the methods in the CodeGuru reference dataset have a lower cyclomatic complexity. This indicates the method has a high number of decisions and it can make the logic difficult to understand and test. We recommend that you simplify this method or break it into multiple methods.